src/modules/expression/expr.rs

```diff
@@ -162,6 +166,8 @@ impl SyntaxModule<ParserMetadata> for Expr {
 impl TranslateModule for Expr {
     fn translate(&self, meta: &mut TranslateMetadata) -> String {
         translate_expression!(meta, self.value.as_ref().unwrap(), [
+            // Builtin
+            Len,
```

Comment:
fyi the order matters in this thing. if its on the bottom it tries to parse that as a VariableGet and throws as error. probably an issue

Reply:
In `translate` this does not matter. This macro expands to a giant `match`:

```
match (module) {
    ...
    Len(item) => item.translate()
    ...
}
```

It matters only in the `parse` function above.